### DISCUSSION TITLE
Bind error on local port 1968 [Address already in use]

### DISCUSSION THREAD

**Discussion by cnmeeia** [omniedgeio/omniedge#17] [General]
ERROR[2022-08-05T20:25:56] Bind error on local port 1968 [Address already in use]
ERROR[2022-08-05T20:25:56] Error: socket setup failed
ERROR[2022-08-05T20:25:56] fail to start omniedge, error info:
 could not start internal

**Comment by yongqianme** [comment]
What command did you run ? 
We can not help you with such limited information.

**Reply by cnmeeia** [reply]
sudo omniedge join -f .omniedge/auth.json

**Reply by yongqianme** [reply]
What is your environment ? 
Would you please check if there is another process using the port ?  Try run: 
```bash
sudo pkill omniedge
```
before join.

**Reply by yongqianme** [reply]
Can you try this version ? I 

[omniedgecli-v0.2.3.1-macos-latest.zip](https://github.com/omniedgeio/omniedge/files/9274095/omniedgecli-v0.2.3.1-macos-latest.zip)
